jobbergate-agent/jobbergate_agent/settings.py

```diff
@@ -50,6 +50,9 @@ class Settings(BaseSettings):
     TASK_JOBS_INTERVAL_SECONDS: int = Field(60, ge=10, le=3600)  # seconds
     TASK_GARBAGE_COLLECTION_HOUR: Optional[int] = Field(None, ge=0, le=23)  # hour of day
 
+    # Job submission settings
+    DOWNLOAD_JOB_SCRIPTS: bool = True
```

Comment:
Maybe it should refer to saving the files on disk? For clarification, because they are downloaded from the backend anyway to send to slurm restd.

Reply:
Renamed it to `WRITE_SUBMISSION_FILES`.